image/bootstrap.sh

```diff
@@ -33,6 +37,13 @@ dpkg -i busybox-static_1.22.0-19+b3_armel.deb
 systemctl disable uuidd.socket
 rm uuid-runtime_2.29.2-1+deb9u1_armel.deb busybox-static_1.22.0-19+b3_armel.deb
 
+# Install RTL8811CU drivers on first boot
+sed -e '/^\s*;;$/{i \
+        depmod\
+        modprobe 8821cu\
```

Comment:
This modprobe seems to be redundant

Reply:
I'll direct this to Prof @boydanderson as well as these instructions were provided by him and I'm not too familiar in this area.

Reply:
I think this can be removed. As you are doing a reboot anyway.

Reply:
Resolved in 784d781cd0d4a1b02a4bc035fe8228db0d3e4f3d.